app/assets/javascripts/components/reviewer/reviewerDashboard.js.jsx

```diff
@@ -4,17 +4,17 @@ class ReviewerDashboard extends React.Component {
     super(props)
 
     this.cohorts = JSON.parse(this.props.cohorts)
-    this.current_cohort = JSON.parse(this.props.current_cohort)
+    if(this.current_cohort){this.current_cohort = JSON.parse(this.props.current_cohort)}
```

Comment:
Maybe I'm misunderstanding but doesn't this line mean this line will never run since `this.current_cohort` is never set? Should this `if` be `if (this. props.current_cohort`?

Reply:
Yes, wow- must've been really late when I made these changes. Thanks for the catches!